test/known_issues/test-http-path-contains-unicode.js

```diff
@@ -10,7 +10,7 @@ const http = require('http');
 
 const expected = '/café🐶';
 
-assert.strictEqual('/caf\u{e9}\u{1f436}', expected);
+assert.strictEqual(expected, '/caf\u{e9}\u{1f436}');
```

Comment:
Are we sure that this one is correct?

Reply:
@cjihrig I think both the code and the change are correct… are you worried about anything in particular?

Reply:
I guess maybe it's an edge case or just poorly named. Either way, I won't block this.